tests/src/Interop/ICustomMarshaler/ConflictingNames/SameNameDifferentAssembly.cs

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.Loader;
+using TestLibrary;
+
+public class RunInALC
+{
+    public static int Main(string[] args)
+    {
+        Assert.AreEqual(123, new CustomMarshalers.CustomMarshalerTest().ParseInt("123"));
```

Comment:
Wrap in try/catch. Tests should properly return error codes rather than crash to indicate failure.